### ISSUE TITLE
For Giving Block projects, add "tweet to ask Gemini to list GIV!" prompt (after launch)

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-next#416]
as per: https://github.com/Giveth/GIVeconomy/issues/52

@willyogo what was the best flow we decided here?

**Comment by laurenluz** [comment]
- Giving Block projects to be on Giveth at launch
- Add GIV token as an option
- When they select GIV - say "this is a Giving Block Project and Gemini doesnt accept GIV tokens"
- Popup w a tweet saying "I want to donate GIV to this project, but we need gemini (tag cameron and tyler) to list GIV"

**Comment by WhyldWanderer** [comment]
Hey @markoprljic .. do you know if there has already been a design made for this user flow?

The idea is that we will have lots of Giving Block projects listed on Giveth, however these projects will not be able to accept GIV tokens because we are not yet listed on Gemini. These projects will have a limited list of tokens that they can accept.

We would like to have GIV as an option in the list but when the user selects GIV as their donation currency, the donate button will become inactive and it will give the user a CTA to tweet about it. The tweet would say something like "Help get GIV listed on Gemini so this project can accept GIV donations!". It would be really cool if we could customize the tweet to include the name of the project they were attempting to donate to like "I want to donate GIV to Save the Children, wen GIV on @gemini ser?"

**Comment by markoprljic** [comment]
@WhyldWanderer No, this isn't in the works yet BUT we can make it happen! Maybe @mosaeedi can work on this and add it to the current Donation flow. You pretty much described it all here so it's a no brainer.

**Comment by laurenluz** [comment]
oh sorry for the crossover friends, I made suggestions in here: https://github.com/Giveth/giveth-next/issues/117#issuecomment-982690876

**Comment by markoprljic** [comment]
It's here https://www.figma.com/file/nVoinu0tgJ565enN5R4WDE/Giveth.io-%26-GIVeconomy?node-id=1907%3A69961

**Comment by MoeNick** [comment]
pls relate your pr in nextjs version. @mateodaza

**Comment by mateodaza** [comment]
![image](https://user-images.githubusercontent.com/18689480/145682874-c36d3582-72be-469f-a76d-b708697a8a52.png)

in staging
